### PR TITLE
Python: Most TestModels

### PR DESCRIPTION
**Pull request by lucasmcdonald3**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: TestModels/Aggregate/runtimes/python/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    { include = "simple_aggregate", from = "src" }
+]
+# Include all of the following .gitignored files in package distributions,
+# even though it is not included in version control
+include = ["**/smithygenerated/**/*.py", "**/internaldafny/generated/*.py"]
+
+[tool.poetry.dependencies]
+python = "^3.11.0"
+DafnyRuntimePython = "^4.4.0"
+# TODO: Depend on PyPi once Smithy-Python publishes their Python package there
```
Comment (kessplas) [inline TestModels/Aggregate/runtimes/python/pyproject.toml:16]: nit: I suppose these comments aren't strictly accurate anymore? If I recall correctly, we plan to maintain a Smithy-Python fork indefinitely?

File: TestModels/Errors/src/WrappedSimpleErrorsImpl.dfy [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 include "../Model/SimpleErrorsTypesWrapped.dfy"
 
-module {:extern "simple.errors.internaldafny.wrapped"} WrappedSimpleErrorsService refines WrappedAbstractSimpleErrorsService {
```
Comment (kessplas) [inline TestModels/Errors/src/WrappedSimpleErrorsImpl.dfy:5]: nit/question: did we decide how we're resolving the lack of extern trait for non-Python (and Go?) runtimes in 4.7?

**Review by kessplas — APPROVED** [review]
2 non-blocking comments, otherwise LGTM

File: TestModels/Aggregate/runtimes/python/pyproject.toml (hunk 1 above)
Comment (lucasmcdonald3) [reply TestModels/Aggregate/runtimes/python/pyproject.toml:16]: Hopefully we don't maintain the fork indefinitely.

This comment is still accurate; if Smithy-Python were to publish their runtime to PyPI we should be able to depend on it immediately

File: TestModels/Errors/src/WrappedSimpleErrorsImpl.dfy (hunk 2 above)
Comment (lucasmcdonald3) [reply TestModels/Errors/src/WrappedSimpleErrorsImpl.dfy:5]: José is actively looking into this. I think the current plan is something like
```
module {:extern } WrappedSimpleErrorsService
```
but we're going to work together to see what makes sense over the next week or so. 

My plan is to have him resolve this on main for Java/NET, then I will pull in whatever the solution is.

**Pull request merged**